src/scss/grommet-core/_objects.drop.scss

```diff
@@ -3,9 +3,19 @@
 .drop {
   position: fixed;
   z-index: 20;
-  background-color: $background-color;
+  background-color: rgba(255, 255, 255, 0.9);
   border: $drop-border;
   border-radius: $border-radius;
   box-shadow: $drop-box-shadow;
   overflow: auto;
-}
+  
+  .menu__control--fixed-label {
+    padding: halve($inuit-base-spacing-unit) 0 $inuit-base-spacing-unit * 0.75;
```

Comment:
This padding signature seems just tad complicated. What are we trying to align with it?

How about trying: display: flex; align-items: center;